nuxeo-page-provider-display-behavior.html

```diff
@@ -514,15 +514,37 @@
       _fetchRange(firstIndex, lastIndex, clear) {
         if (this._hasPageProvider()) {
 
+          if (firstIndex === 0) {
+            lastIndex = this._pageSize - 1;
+          }
+
           if (!firstIndex && !lastIndex && !clear) {
```

Comment:
Given that firstIndex maybe `0`, which evaluates to `false`, and that clear may also be `false`, shouldn't we be comparing these to `undefined`?

Reply:
This part, I am not even sure why it is for.. The way I see it, firstIndex and lastIndex are never meant to be undefined. So in case first and last index both equal 0 despite clear is false, we force clear/reinit